### PR TITLE
Change errors to warnings

### PR DESCRIPTION
**Pull request by daniel-thom**

### CODE REVIEW

File: src/utils/IO/system_checks.jl [hunk 1]
```diff
@@ -113,7 +109,12 @@ Total sum of system generator ratings.
 - `sys::System`: system
 """
 function total_generation_rating(sys::System)
```
Comment (jd-lara) [inline src/utils/IO/system_checks.jl:111]: we should add storage here

Comment (daniel-thom) [reply src/utils/IO/system_checks.jl:111]: Should the name of this function change?

Comment (jd-lara) [reply src/utils/IO/system_checks.jl:111]: total_capacity_rating

Comment (claytonpbarrows) [reply src/utils/IO/system_checks.jl:111]: Ok, I support adding storage and changing to total_capacity_rating

**Pull request merged**